Gruntfile.js

```diff
@@ -179,7 +179,11 @@ module.exports = function (grunt) {
     wiredep: {
       app: {
         src: ['<%= yeoman.app %>/index.html'],
-        exclude: ['bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js'],
+        exclude: [
+            'bower_components/bootstrap-sass-official/assets/javascripts/bootstrap.js',
+            // Leaflet provided by cartodb.js
+            'bower_components/leaflet-dist/*'
```

Comment:
BTW, it might be good to check on adding Leaflet plugins if their bower depends on `leaflet-dist` or `leaflet` (main); they vary.  It might even be a good idea to go ahead and ignore `leaflet` here too, now, to avoid any future proliferation of Leaflets.